MediaCV.css

```diff
@@ -1,100 +0,0 @@
-/* media queries */
-
-/********BREAKPOINTS********
-Moyenne résolution = 1024px
-Petite résolution = 767px
-****************************/
-
-
-@media screen and (max-width: 1024px)
```

Comment:
Pourquoi utilises-tu 

```css
@media screen and (max-width: 1024px)
```

et pas 

```css
@media (max-width: 1024px)
```

?

Reply:
Dans le cours HTML/CSS d'Openclassroom, il utilise @media screen and (max-width: 1024px).